src/lib/helpers/updateNotifier.js

```diff
@@ -93,7 +93,7 @@ class UpdateNotifier {
 
   async fetchInfo () {
     const { distTag } = this.options
-    const latest = await latestVersion(this.packageName, distTag)
+    const latest = await latestVersion(this.packageName, distTag) ?? this.packageVersion
```

Comment:
does this `??` work with node 16? the binary gets built with node 16.

Reply:
yep, been there since node 14 iirc